internal/cli/iam/users/invite_test.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package users
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/mocks"
+	"go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+func TestInvite_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockUserCreator(ctrl)
+	defer ctrl.Finish()
+
+	expected := &mongodbatlas.AtlasUser{
+		Username: "testUser",
+	}
+	opts := &InviteOpts{
+		store:       mockStore,
+		country:     "",
+		mobile:      "",
+		email:       "",
+		lastName:    "",
+		firstName:   "",
+		password:    "",
```

Comment:
[nit] you can ommit this, not setting them or setting them to empty string is the same